### DISCUSSION TITLE
waha with autoreply messages on whatssap

### DISCUSSION THREAD

**Discussion by mankhokwe** [devlikeapro/waha#431] [Q&A]
how do  you achieve autoreply messages on whatsapp using devlikeapro/waha 
like setting some messages once your chart recievies a message waha should reply back

**Comment by devlikepro** [comment]
hi! For that you need to use two things:
1. Listen to `message` webhook event https://waha.devlike.pro/docs/how-to/webhooks/#message
2. Answer to the chat with the message using `POST /api/sendText` https://waha.devlike.pro/docs/how-to/send-messages/#send-text

Check the python example here, it's basically do what you're asking!
https://waha.devlike.pro/docs/examples/python/
https://github.com/devlikeapro/waha/blob/core/examples/python/whatsapp_echo_bot.py 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
